database/migrations/2024_05_27_105458_create_tasks_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('tasks', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('todo_id')
+                ->constrained()
+                ->cascadeOnUpdate()
+                ->cascadeOnDelete();
+            $table->string('title');
+            $table->string('description')->nullable();
+            $table->boolean('state')->default(false);
```

Comment:
If this is the task activity flag, then it should be used  "is_active". If there can be several states, then use int